app/views/physical_ticket/show.pdf.prawn

```diff
@@ -0,0 +1,64 @@
+prawn_document(filename: @file_name, page_layout: :portrait, :page_size =>'A4' ) do |pdf|
+  # Vertical Layout
+  user = @physical_ticket.user
+  ticket_purchase = @physical_ticket.ticket_purchase
```

Comment:
same with `ticket`, `organization` and `venue`. `user` is used twice so I am fine with leaving it.